grammar.js

```diff
@@ -37,6 +37,7 @@ module.exports = grammar(HTML, {
     '#',
     '/',
     ':',
+    '|',
```

Comment:
I don't think `|` needs to be an external, since this PR doesn't include any changes to `scanner.c`. If the tests pass after it's removed, I'd say leave it out.

Reply:
I will check if this is required in zed to highlight that delimiter, if not, I will remove it, but if it is required, I think it should stay

Reply:
No special snowflake favors for editors ;) Either something works for the language, or it doesn't. Editor-specifics should be handed (if at all!) by queries.

Reply:
Sorry what I mean is that any editor should be able to query on that delimiter in order to be able to highlight it

Not strictly zed specific, but just in general, it should be possible to style all the various nodes

Reply:
Works. My mistake!

thought that had to be there to enable delimiter highlighting